src/lib.rs

```diff
@@ -23,25 +23,43 @@ use std::io;
 pub struct Metadata {
     /// A list of all crates referenced by this crate (and the crate itself)
     pub packages: Vec<Package>,
-    resolve: Option<()>,
+    /// Dependencies graph
+    pub resolve: Option<Resolve>,
```

Comment:
Well. The field was private, so it's only a small breaking change since the parsed json is tested more strictly. I think it should be a patch version bump, since no real code should break (it should be valid metadata json to begin with)